spec/lib/object_serializer_class_methods_spec.rb

```diff
@@ -16,35 +16,51 @@
     end
 
     context 'with namespace' do
+
```

Comment:
The only material change in this file is that the serializer classes are actually defined.